### PR TITLE
Update attributes-rating.md

### PR DESCRIPTION
**Pull request by pawankparmar**
## Purpose of this pull request

1. Path for rating was wrong.
2. Added "Is Active" checkbox working.

## Affected documentation pages

- https://docs.magento.com/m2/ee/user_guide/stores/attributes-rating.html
- https://docs.magento.com/m2/ce/user_guide/stores/attributes-rating.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by pawankparmar** [comment]
It is fixed for issue issue #237

File: src/stores/attributes-rating.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ _Ratings_
 
 1. For **Sort Order**, enter a number to determine the order of this rating when listed with others.
 
+1. If you want to show your rating on front end, tick the **Is Active** checkbox.
```
Comment (leslietilling) [inline src/stores/attributes-rating.md]:
```suggestion
1. If you want to show your rating on the storefront, select the **Is Active** checkbox.
```
We use select/deselect for checkboxes.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**